.eslintrc.cjs

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+module.exports = {
+    root: true,
+    extends: ['scratch', 'scratch/es6', 'scratch/react', 'plugin:import/errors'],
+    env: {
+        node: true,
+    },
+    rules: {
+        'import/no-commonjs': 'error',
+        'react/jsx-no-literals': 'error',
+        'react/jsx-filename-extension': [
+            'error',
+            {
+                extensions: ['.jsx', '.tsx']
+            }
+        ],
+        'import/no-mutable-exports': 'error',
+        'import/no-amd': 'error',
+        'no-confusing-arrow': ['error', {
+            allowParens: true
+        }],
+        'comma-dangle': 'off',
+    },
+    settings: {
+        'import/extensions': [
+            '.cjs',
+            '.js',
+            '.jsx',
+            '.mjs',
+            '.ts',
+            '.tsx',
+        ],
+        'import/resolver': {
+            webpack: {
+                config: path.resolve('webpack.config.js')
+            }
+        },
+        'react': {
+            version: '16.2' // Prevent 16.3 lifecycle method errors
```

Comment:
Previously, we only specified the React version in `src/`, which meant that using the React plugin for eslint in `test/` would generate a big warning.